scripts/common/_rid.sh

```diff
@@ -23,7 +23,7 @@ if [ -z "$RID" ]; then
             export RID=ubuntu.14.04-x64
         elif [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
             export OSNAME=centos
-            export RID=centos.7-x64
+            export RID=rhel.7-x64
```

Comment:
This one shouldn't need to be changed, it's given to NuGet and NuGet knows what to do.